src/library/rekordbox/rekordboxfeature.h

```diff
@@ -0,0 +1,117 @@
+// rekordboxfeature.h
+// Created 05/24/2019 by Evan Dekker
+
+// This feature reads tracks, playlists and folders from removable Recordbox 
+// prepared devices (USB drives, etc), by parsing the binary *.PDB files
+// stored on each removable device. It does not read the locally stored
+// Rekordbox database (Collection).
+
+// It draws heavily from the hard work completed here:
+
+//      https://github.com/Deep-Symmetry/crate-digger
+
+// And uses the C++ Kaitai Struct binary parsing libraries:
+
+//      http://kaitai.io
+//      https://github.com/kaitai-io/kaitai_struct
+//      https://github.com/kaitai-io/kaitai_struct_cpp_stl_runtime
+
+// The *.PDB C++ files:
+
+//      rekordbox_pdb.h
+//      rekordbox_pdb.cpp
+
+// Were generated from the following structure definition file:
+
+//      https://github.com/Deep-Symmetry/crate-digger/blob/master/src/main/kaitai/rekordbox_pdb.ksy
+
+#ifndef REKORDBOX_FEATURE_H
+#define REKORDBOX_FEATURE_H
+
+#include <QStringListModel>
+#include <QtSql>
+#include <QXmlStreamReader>
+#include <QFuture>
+#include <QtConcurrentRun>
+#include <QFutureWatcher>
+
+#include <fstream>
+
+#include "library/baseexternallibraryfeature.h"
+#include "library/baseexternaltrackmodel.h"
+#include "library/baseexternalplaylistmodel.h"
+#include "library/treeitemmodel.h"
+
+#include "library/rekordbox/rekordbox_pdb.h"
+
+#define PDB_PATH "PIONEER/rekordbox/export.pdb"
```

Comment:
Is this really needed in the header file? Define as a constant in an anonymous namespace in the .cpp file.